### DISCUSSION TITLE
Add support for NextJS

### DISCUSSION THREAD

**Discussion by IRediTOTO** [windicss/windicss#250] [Ideas]
Hi NextJS is very popular. Can you please make a support for this framework. I really like WindiCSS.

**Comment by shiftgeist** [comment]
Nextjs is a little unknown to me but looking at the docs it is webpack based and you can add the [windicss-webpack-plugin](https://github.com/windicss/windicss-webpack-plugin) inside a [custom webpack config](https://nextjs.org/docs/api-reference/next.config.js/custom-webpack-config) for nextjs. TLDR it seems supported.
